docs/developer/app-store/apps/taxes.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Taxes
+sidebar_position: 2
+---
+
+# Taxes
+
+## Overview
+
+_Taxes_ is a Saleor app that allows delegating tax calculations to external tax providers. Here are the currently available providers:
+
+- [TaxJar](https://www.taxjar.com/)
+- [Avatax](https://www.avalara.com/us/en/products/calculations.html)
+- [Avatax Excise](https://developer.avalara.com/excise/) [_coming soon_]
+
+Taxes App has two views: **Channels** and **Providers**.
+
+**Channels** allows you to fill in the "ship from" address and assign a tax provider instance to the channel.
+
+**Providers** allows you to manage the tax provider instances. You can have multiple instances of the same provider.
```

Comment:
I'm not sure if this is a good approach, to tightly couple how app looks to the docs, it will make docs outdated soon.

I don't think its important to have "views" here. More important is what are the features, what can be done, what are assumptions and limitations. You can visit other platforms appstores for reference.

I suggest to mention:
- App allows to create more than one configuration for each provider (you can test with Dev and Prod instances of Taxjar or Avalara if you like)
- App allows you to select specific provider per channel. Its a great opportunity to mix providers for specific markets
- Visit app built-in instructions for usage details 


I think its far more accurate in readme, so people know what to expect.
App itself contains instructions and views/tabs are self explaining - we are building a product for developers